app/templates/pdf/order_invoice.html

```diff
@@ -136,7 +136,7 @@ <h3 style="text-align:center;">{{ ("Order Invoice") }}</h3>
                                     {{ ('NA') }}
                                 {% endif %}<br>
                         </td>
-                        {% if order.is_billing_enabled %}
+                        {% if has_billing_info %}
```

Comment:
Just a question, why do we need a check? If order.billing_info is serving the purpose

Reply:
New paid orders aren't using `is_billing_enabled` anymore, that's why .

Reply:
@uds5501 can't we proceed to make is_billing_enabled true is order isn't free either on server or on fron so that we can maintain the consistency of data? I think this will be a better approach because we can anytime get if a order has billing info or not just by a single field.

Reply:
@shreyanshdwivedi understood, fixing this